core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala

```diff
@@ -99,27 +93,27 @@ private[spark] class HadoopPartition(rddId: Int, idx: Int, @transient s: InputSp
  */
 @DeveloperApi
 class HadoopRDD[K, V](
+  sc: SparkContext,
```

Comment:
This indentation is incorrect now. I would not change imports/indentation unless you know it's wrong